contrib/nix/flake.nix

```diff
@@ -139,11 +188,13 @@
             src = fileSetForCrate ../..;
           }
         );
+
+        treefmt = treefmt-nix.lib.evalModule pkgs (import ./treefmt.nix);
       in
       with pkgs;
       {
         packages = {
-          inherit stacks-signer;
+          inherit stacks-signer stacks-common cargoArtifacts stacks-core;
```

Comment:
I think you can remove `cargoArtifacts`, I added it jus to test building it quickly.